frontend/packages/dev-console/src/components/monitoring/alerts/monitoring-alerts-utils.tsx

```diff
@@ -0,0 +1,127 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+import { expandable, sortable, cellWidth, SortByDirection } from '@patternfly/react-table';
+import {
+  StateCounts,
+  Severity,
+  AlertState,
+  alertsRowFilters,
+  severityRowFilter,
+} from '@console/internal/components/monitoring';
+import { Kebab } from '@console/internal/components/utils';
+import {
+  alertDescription,
+  alertState,
+  alertSeverityOrder,
+  alertingRuleStateOrder,
+} from '@console/internal/reducers/monitoring';
+import { Alert, Rule } from '@console/internal/components/monitoring/types';
+import { YellowExclamationTriangleIcon } from '@console/shared';
+
+const viewAlertRule = {
+  label: 'View Alerting Rule',
+};
+
+export const monitoringAlertColumn = [
+  {
+    title: 'Name',
+    cellFormatters: [expandable],
+    transforms: [sortable],
+  },
+  { title: 'Severity', transforms: [sortable, cellWidth(20)] },
+  { title: 'Alert State', transforms: [sortable, cellWidth(20)] },
+  '',
+];
+
+export const monitoringAlertRows = (alertrules: Rule[]) => {
+  const rows = [];
+  _.forEach(alertrules, (rls) => {
+    rls.type === 'alerting' &&
```

Comment:
I now realize that `getAlertsAndRules` filters out all alerts that are not of type `alerting`, so actually we don't need this `rls.type === 'alerting'` check.